src/NosCore.Controllers/CharacterScreenPacketController.cs

```diff
@@ -315,6 +318,15 @@ public void SelectCharacter(SelectPacket selectPacket)
                     Session.Character.Mp = (int) Session.Character.MPLoad();
                 }
 
+                IEnumerable<CharacterRelationDTO> relations = DAOFactory.CharacterRelationDAO.Where(s => s.CharacterId == Session.Character.CharacterId);
+                List<CharacterDTO> characters = DAOFactory.CharacterDAO.Where(s => relations.Select(v => v.RelatedCharacterId).Contains(s.CharacterId)).ToList();
```

Comment:
can you manually add something in db and check this doesnt crash ?

Reply:
I've tested it, it works fine

Reply:
was just to make sure linq to sql was enable to handle it